src-java/opentsdb-topology/opentsdb-storm-topology/src/test/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopologyTest.java

```diff
@@ -123,7 +124,7 @@ public void shouldSendDatapointRequestsTwice() throws Exception {
             // This test expects to see 2 POST requests to OpenTsdb, but if batch.size > 1 OtsdbBolt will send
             // 1 request with 2 metrics instead of 2 requests with 1 metric.
             // So next property forces OtsdbBolt to send 2 requests.
-            Properties properties = new Properties();
+            Properties properties = getProperties();
             properties.put("opentsdb.batch.size", "1");
```

Comment:
Why not move this line into getProperties method too?

Reply:
Because we need this option only in one test. It's workaround to get 2 requests for opentsdb. Other tests should run with default parameters to test topology in common case.